README.building.md

```diff
@@ -2,18 +2,20 @@
 
 Install junit 4, Apache ant and Hamcrest. For Debian Wheezy these are `junit4`, `ant`, `ant-optional` and `libhamcrest-java`.
 
-If sufficient packages are not available, put these jars into the `lib/` directory:
+Freenet also requires `freenet-ext.jar` in the `lib/freenet/` directory. To get it, [download](https://downloads.freenetproject.org/latest/freenet-ext.jar) or copy from the freenet folder of an existing installation or build the `contrib/` submodule.
 
-* `junit4.jar` and `hamcrest-core.jar`: See the [download instructions](https://github.com/junit-team/junit/wiki/Download-and-Install)
-* `bcprov-jdk15on-152.jar`: Download Bouncy Castle 1.52: `wget --no-passive-ftp ftp://ftp.bouncycastle.org/pub/release1.52/bcprov-jdk15on-152.jar` or `wget http://www.bouncycastle.org/download/bcprov-jdk15on-152.jar`
+Then just execute `ant`.
 
-Freenet also requires in the `lib/freenet/` directory:
+You can override values of variables set in build.xml by putting them into the file override.properties in the format `variable = value`. To build Freenet without running the tests, use `ant -Dtests.skip=true`.
 
-* `freenet-ext.jar`: Build the `contrib/` submodule or [download](https://downloads.freenetproject.org/latest/freenet-ext.jar) for convenience.
 
-The dependencies could also be copied from an existing Freenet installation.
+If sufficient packages for the dependencies are not available, put these jars into the `lib/` directory:
 
-## Building Freenet from source: Using Eclipse etc
+* `junit4.jar` and `hamcrest-core.jar`: See the [download instructions](https://github.com/junit-team/junit/wiki/Download-and-Install)
```

Comment:
This is horrible. Dependencies should always be listed with _exact_ version information, otherwise developers will have a very shitty time trying to track down the correct versions.